src/Aspire.Hosting.NodeJs/NodeAppResource.cs

```diff
@@ -12,4 +12,4 @@ namespace Aspire.Hosting;
 /// <param name="command">The command to execute.</param>
 /// <param name="workingDirectory">The working directory to use for the command. If null, the working directory of the current process is used.</param>
 public class NodeAppResource(string name, string command, string workingDirectory)
-    : ExecutableResource(name, command, workingDirectory), IResourceWithServiceDiscovery;
+    : ExecutableResource(name, command, workingDirectory), IResourceWithServiceDiscovery, IResourceWithStaticDockerFiles;
```

Comment:
I'm not sure this is a good change. I question if we need an interface at all, TBH. Node apps don't always produce static files. They might be their own server all together.

Reply:
Today `AddViteApp` returns `IResourceBuilder<NodeAppResource>`. So maybe when we pull it into the repo, we make it derived from `NodeAppResource`?